tests/x11/reboot_gnome.pm

```diff
@@ -40,12 +40,16 @@ sub run() {
 
         send_key "ret";
 
-        wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
-        if (check_screen 'reboot-auth', 2) {
-            record_soft_failure 'bsc#981299';
-            send_key_until_needlematch 'generic-desktop', 'esc',             7, 10;    # close timed out authentication window
-            send_key_until_needlematch 'logoutdialog',    'ctrl-alt-delete', 7, 10;    # reboot
-            assert_and_click 'logoutdialog-reboot-highlighted';
+        # workaroung for bsc#981299
+        # run only on qemu backend, on svirt check_screen still get last image even if system is already down
```

Comment:
we need a poo for that - we can't simply accept that. It's a bug that will hit us in more places